x-pack/solutions/security/plugins/security_solution/public/explore/network/components/embeddables/map_config.ts

```diff
@@ -290,75 +315,79 @@ export const getLineLayer = (
   indexPatternId: string,
   parentId: string,
   layerDetails: LayerMapping
-) => ({
-  sourceDescriptor: {
-    type: SOURCE_TYPES.ES_PEW_PEW,
-    applyGlobalQuery: true,
-    id: uuidv4(),
-    indexPatternId,
-    sourceGeoField: layerDetails.source.geoField,
-    destGeoField: layerDetails.destination.geoField,
-    metrics: [
-      {
-        type: 'sum',
-        field: layerDetails.source.metricField,
-        label: layerDetails.source.metricField,
-      },
-      {
-        type: 'sum',
-        field: layerDetails.destination.metricField,
-        label: layerDetails.destination.metricField,
-      },
-    ],
-  },
-  style: {
-    type: 'VECTOR',
-    properties: {
-      fillColor: {
-        type: 'STATIC',
-        options: { color: '#1EA593' },
-      },
-      lineColor: {
-        type: 'STATIC',
-        options: { color: euiVisColorPalette[1] },
-      },
-      lineWidth: {
-        type: 'DYNAMIC',
-        options: {
-          field: {
-            label: 'count',
-            name: 'doc_count',
-            origin: 'source',
-          },
-          minSize: 1,
-          maxSize: 8,
-          fieldMetaOptions: {
-            isEnabled: true,
-            sigma: 3,
+) => {
+  const euiVisColorPalette = euiPaletteColorBlind();
+
+  return {
+    sourceDescriptor: {
+      type: SOURCE_TYPES.ES_PEW_PEW,
+      applyGlobalQuery: true,
+      id: uuidv4(),
+      indexPatternId,
+      sourceGeoField: layerDetails.source.geoField,
+      destGeoField: layerDetails.destination.geoField,
+      metrics: [
+        {
+          type: 'sum',
+          field: layerDetails.source.metricField,
+          label: layerDetails.source.metricField,
+        },
+        {
+          type: 'sum',
+          field: layerDetails.destination.metricField,
+          label: layerDetails.destination.metricField,
+        },
+      ],
+    },
+    style: {
+      type: 'VECTOR',
+      properties: {
+        fillColor: {
+          type: 'STATIC',
+          options: { color: '#1EA593' },
```

Comment:
I think we can replace this hard coded color with euiVisColorPalette[1] ( the same as below)